lib/system/api/containers.toit

```diff
@@ -39,19 +39,18 @@ interface ContainerService:
   image-writer-commit handle/int flags/int data/int -> uuid.Uuid
   static IMAGE-WRITER-COMMIT-INDEX /int ::= 5
 
-  // INDEX 8 is reserved for ContainerMessageService so that
-  // a provider can implement both ContainerService and ContainerMessageService
+  // Indexes 100+ are reserved for ContainerEventService so that
+  // a provider can implement both ContainerService and ContainerEventService
   // in the same class.
 
-interface ContainerMessageService:
+interface ContainerEventService:
```

Comment:
I'm still not convinced that it makes sense to split this into two separate APIs.